### ISSUE TITLE
Package does not work for decks created with the latest Anki version

### ISSUE THREAD

**Issue by Torvaney** [zfletch/apkg-to-csv#3]
Hello,

I've used your tool for a couple of decks and it's worked great. Thank you for developing and sharing it.

However, I recently tried to use it for [this deck](https://ankiweb.net/shared/info/638411848) and got the following output:

```bash
> bin/apkg-to-csv ~/Downloads/Ultimate_Spanish_Conjugation_Lisardos_KOFI_Method.apkg

Text,Back Extra

Front,Back,Add Reverse

Front,Back

Front,Back
"Please update to the latest Anki version, then import the .colpkg/.apkg file again."

Front,Back
```

I'm not too familiar with Ruby or the Anki data format, but this deck appears to have a slightly different format to the other ones I converted successfully. When unzipped, the successful files have a `media` and a `collection.anki2` file; whereas the unsuccessful file has additional `meta` and `collection.anki21` files.

When I open the `collection.anki2` database, the `notes` table has only 1 row:

| id            | guid       | mid           | mod        | usn | tags | flds                                                                                 | sfld                                                                                | csum      | flags | data |
|---------------|------------|---------------|------------|-----|------|--------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------|-----------|-------|------|
| 1692133127934 | GYm3qFoJoE | 1692133127933 | 1692133127 |  -1 |      | Please update to the latest Anki version, then import the .colpkg/.apkg file again. | Please update to the latest Anki version, then import the .colpkg/.apkg file again. | 871165352 |     0 |      |

On the other hand, the `notes` table in `collection.anki21` looks more like what is expected.

So, for the time being, I have successfully worked around this by modifying `bin/apkg-to-csv` to look for `collection.anki21` if it exists and re-installing.

**Comment by zfletch** [comment]
Thanks for creating this issue. I'll try to fix it as soon as I can.

It looks like the issue is that Anki sometimes saves a `collection.anki21` file instead of `collection.anki2`: https://github.com/ankitects/anki/blob/main/pylib/anki/importing/apkg.py#L25. Oddly, that line was added in 2018, so I would have expected this issue to have come up before. Regardless, it seems like we should check for a `collection.anki21` file and use that if it's available.

**Comment by zfletch** [comment]
This should be fixed in the newest release (2.1.0).
